### DISCUSSION TITLE
Der riese v3

### DISCUSSION THREAD

**Discussion by glitcheking** [nzp-team/nzportable#465] [Map Releases]
Der riese v3 pc version

i fixed the zombies and the dogs its fully playeble now removed all the glitches  and added the der riese easter egg can you solve it

Download 
[https://archive.org/download/der_riese_v3/der_riese_v3.rar](url)

![Uploading 20230825_132037.jpg…]()

Made by glitch

**Comment by ToonLink0001** [comment]
ok thanks so much!

**Comment by arynrosh** [comment]
could you reupload link?

**Comment by TheBetaTester000000** [comment]
yo the link is dead really wanted to try this with friends i know its old but. anyway i could get this?
